### PR TITLE
[Bug] - Fixed Copy Icon scrolling horizontally

### LINKED ISSUE #867: The copy button moves if the text box contains a horizontal slider
### What happened?

image

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <adikrish@redhat.com>

This PR Fixes #867 

**Screen recording:**

[Issue867.webm](https://user-images.githubusercontent.com/23498248/186358056-c1a65706-03fd-442c-896e-800c91bd149d.webm)

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) - Yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by lahabana** [comment]
@kleinfreund can you please review this?

File: docs/.vuepress/theme/styles/custom/base/_type-and-content.scss [hunk 1]
```diff
@@ -124,6 +124,15 @@ div[class~='language-yaml']:before {
...
   height: 0;
+  .hover{
+    // Used important as it has to override the inline style from vuepress-plugin-code-copy
+    bottom: 0px !important;
+
+    // Used to fix the overlap of the scroll bar as due to the
+    // above property and hence we set it manually below
+    margin-bottom: 25px !important;
+    margin-right: 15px !important;
+  }
```
Comment (kleinfreund) [inline docs/.vuepress/theme/styles/custom/base/_type-and-content.scss:135]: Please remove the unit from the `0px` value and the `!important` from the margin property declarations as it’s not needed there.

```suggestion
  .hover {
    // Used important as it has to override the inline style from vuepress-plugin-code-copy
    bottom: 0 !important;

    // Used to fix the overlap of the scroll bar as due to the
    // above property and hence we set it manually below
    margin-bottom: 25px;
    margin-right: 15px;
  }
```

**Review by kleinfreund — COMMENTED** [review]
Just a few minor suggestions from my side. Otherwise, this looks good to me.

File: docs/.vuepress/theme/styles/custom/base/_type-and-content.scss (hunk 1 above)
Comment (adithyaakrishna) [reply docs/.vuepress/theme/styles/custom/base/_type-and-content.scss]: Done dona done ✅

**Review by lahabana — APPROVED** [review]
Thanks a lot for this contrib!

**Comment by adithyaakrishna** [comment]
@lahabana I missed updating the space next to :hover, could you please re-review this MR?

**Review by slonka — APPROVED** [review]
That's fine, thanks for the contribution!

**Pull request merged**